post3/runner.py

```diff
@@ -1,14 +1,15 @@
 """
 The main file that will evolve into our trading library
 """
-import v20
 from datetime import datetime, timedelta
+import v20
+
 
-OANDA_ACCESS_TOKEN = "YOUR_ACCESS_TOKEN"
-OANDA_ACCOUNT_ID = '11111' #put your access id here 
+OANDA_ACCESS_TOKEN = "01d4e6de4fc1246d2bfde02450871fe6-0ae159e87f2cc523a9116d9b535e85fa"
```

Comment:
Are you sure you want this here?

Reply:
Thanks for reviewing @elsteelbrain ! 

I should have used a .config file so people don't publish their keys online!